services/github/github-milestone-detail.service.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { GithubAuthV3Service } = require('./github-auth-service')
+const { documentation, errorMessagesFor } = require('./github-helpers')
+
+const schema = Joi.object({
+  open_issues: nonNegativeInteger,
+  closed_issues: nonNegativeInteger,
+  title: Joi.string(),
```

Comment:
Is a title always expected for a milestone? If that's the case, I would suggest adding `.required()` here, otherwise it's fine!

Reply:
You're right, done here and also for state in gh-milestone.